sunkit_image/radial.py

```diff
@@ -658,33 +654,26 @@ def rhef(
     Parameters
     ----------
     smap : `sunpy.map.Map`
-        The sunpy map to enhance.
-    radial_bin_edges : `astropy.units.Quantity`
-        A two-dimensional array of bin edges of size ``[2, nbins]`` where ``nbins`` is
-        the number of bins.
+        The SunPy map to enhance using the RHEF algorithm.
+    radial_bin_edges : `astropy.units.Quantity`, optional
+        A two-dimensional array of bin edges of size ``[2, nbins]`` where ``nbins`` is the number of bins.
+        These define the radial segments where filtering is applied. If None, radial bins will be generated automatically.
     application_radius : `astropy.units.Quantity`, optional
-        The RHEF is applied to emission at radii above the application_radius.
+        The radius above which to apply the RHEF. Only regions with radii above this value will be filtered.
         Defaults to 0 solar radii.
-    upsilon : None, float, or tuple of `float`, optional
-        A double-sided gamma function applied to the equalized histograms.
-        See Equation (4.15) in the thesis.
-        Defaults to 0.35.
-    method : str
-    vignette: `astropy.units.Quantity`, optional
-        Set pixels above this radius to black.
-        Defaults to ``1.5*u.R_sun``.
-        If you want to disable this, pass in None.
-        Set pixels above this radius to black.
-        Defaults to None which is no vignette.
-        One suggested value is ``1.5*u.R_sun``.
-    progress: bool, optional
-        Display a progressbar on the main loop.
-        Defaults to True.
+    upsilon : float or None, optional
+        A double-sided gamma function to apply to modify the equalized histograms. Defaults to 0.35.
+    method : str, optional
+        Method used to rank the pixels for equalization. Defaults to 'inplace'.
+    vignette : `astropy.units.Quantity`, optional
+        Radius beyond which pixels will be set to black. Defaults to ``1.5*u.R_sun``. Set to None to disable vignetting.
```

Comment:
```suggestion
        Radius beyond which pixels will be set to black. Defaults to None, must be in units that are compatible with "R_sun" as the value will be transformed. 
```

Reply:
I'm thinking that setting to NAN is probably more useful since the values aren't valid out there anyway.

Reply:
Sure but you won't pass nan to the keyword?

Reply:
No, I was just confused.